test/operator/test_local_operator.py

```diff
@@ -318,7 +318,7 @@ def test_copy():
         assert op_copy is not op
         for o1, o2 in zip(op._operators, op_copy._operators):
             assert o1 is not o2
-            assert np.all(o1 == o2)
+            assert np.all(o1.A == o2.A)
```

Comment:
what is `.A` ?

Reply:
It's accessing the dense representation. I could use `.todense()` if you want to. I just thought this might be more general because a dense array also has the `.A` property

Reply:
no that's fine. I just had never seen this

Reply:
I added the comment

Reply:
Great!
When you address those items please click on ‘resolve conversation’.
It helps me review faster…